src/js/operations/GeoExecute.js

```diff
@@ -54,25 +54,16 @@ L.DNC.GeoExecute = L.Class.extend({
 
         // if the new object is a feature collection and only has one layer,
         // remove it and just keep it as a feature
-        if ( newLayer.geometry.type == "FeatureCollection" &&
-             newLayer.geometry.features.length == 1 ) newLayer.geometry = this._unCollect( newLayer.geometry );
-
-        var mapLayer = L.mapbox.featureLayer(newLayer.geometry);
```

Comment:
This type of logic is removed from this class.  `GeoExecute` isn't in charge of doing anything with the data, it just executes the operation and returns the results.  This should make testing a bit easier.